pages/devel-version.html

```diff
@@ -157,22 +157,38 @@ <h5>Quickstart Installation Instructions</h5>
                 <p>If you want to install Uyuni in lab mode, for testing, you may be interested in how to easily make use of libvirt's included dnsmasq DNS and DHCP server: <a href="https://github.com/uyuni-project/uyuni/wiki/Libvirt-DNS-and-DHCP-without-Avahi">Libvirt DNS and DCHP without Avahi</a>.</p>
 
                 <p>1. Update openSUSE Leap 15.5:</p>
-                <code>$ sudo zypper update</code>
+                <code>$ transactional update</code>
 
-                <p>2. Configure a resolvable fully quallified domain name with <strong>Yast2 > System > Netwok Settings > Hostname/DNS</strong></p>
+                <p>2. Reboot the system</p>
 
-                <p>3. Add the repository for the server:</p>
-                <code>$ sudo zypper ar https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master/images/repo/Uyuni-Server-POOL-$(arch)-Media1/ uyuni-server-devel</code>
+                <p>3. Configure a resolvable fully quallified domain name with <strong>Yast2 > System > Netwok Settings > Hostname/DNS</strong></p>
 
-                <p>4. Refresh metadata from the repositories:</p>
-                <code>$ sudo zypper ref</code>
+                <p>4. Enter the transactional shell:</p>
+                <code>$ transactional-update shell</code>
```

Comment:
Don't you need this a root, as well as the other commands?
```suggestion
                <code># transactional-update shell</code>
```